master/pkg/schemas/expconf/storage_config.go

```diff
@@ -174,19 +174,20 @@ type AzureConfigV0 struct {
 
 // Merge implements schemas.Mergeable.
 func (c AzureConfigV0) Merge(other interface{}) interface{} {
-	out := AzureConfigV0{}
 	otherConfig := other.(AzureConfigV0)
-	out.RawContainer = schemas.Merge(c.RawContainer, otherConfig.RawContainer).(*string)
 	var credSource AzureConfigV0
 	if c.RawConnectionString != nil || c.RawAccountURL != nil {
 		credSource = c
 	} else {
 		credSource = otherConfig
 	}
-	out.RawConnectionString = schemas.Copy(credSource.RawConnectionString).(*string)
-	out.RawAccountURL = schemas.Copy(credSource.RawAccountURL).(*string)
-	out.RawCredential = schemas.Copy(credSource.RawCredential).(*string)
-	return out
+
+	return AzureConfigV0{
```

Comment:
prob could do more like this, in a way, writing the merge like this with this linter makes sure it stays complete.